package.json

```diff
@@ -56,15 +56,13 @@
     "@types/async-retry": "^1.4.3",
     "@types/configstore": "^5.0.0",
     "@types/extend": "^3.0.1",
-    "@types/is-stream": "^1.1.0",
     "@types/mocha": "^8.0.0",
     "@types/mockery": "^1.4.29",
     "@types/node": "^16.0.0",
     "@types/pumpify": "^1.4.1",
     "@types/sinon": "^10.0.0",
     "c8": "^7.0.0",
     "gts": "^2.0.0",
-    "is-stream": "^2.0.0",
```

Comment:
Out of curiosity, what did this do and why is it safe for removal?

Reply:
It was used to validate if an object is a stream in tests. It’s no longer needed.